hack/integration-cli-on-swarm/host/host.go

```diff
@@ -105,22 +106,21 @@ func xmain() error {
 		logrus.Infof(" - Worker image: %s", workerImageForStack)
 	}()
 	if err = deployStack(cli, defaultStackName, compose); err != nil {
-		return err
+		return 1, err
 	}
 	logrus.Infof("The log will be displayed here after some duration."+
 		"You can watch the live status via `docker service logs %s_worker`",
 		defaultStackName)
 	masterContainerID, err := waitForMasterUp(cli, defaultStackName)
 	if err != nil {
-		return err
+		return 1, err
 	}
 	rc, err := waitForContainerCompletion(cli, os.Stdout, os.Stderr, masterContainerID)
 	if err != nil {
-		return err
+		return 1, err
```

Comment:
I think the current code is ok, because when `xmain` returns non-nil `err`, `main` always ignores `rc` and exits with status 1.